### DISCUSSION TITLE
Could we view the icon of 16bit executive again?

### DISCUSSION THREAD

**Discussion by BUGProtectionAssociation** [otya128/winevdm#1278] [Ideas]
Somebody knows that the [icon supports of 16bit has end from Vista](https://learn.microsoft.com/en-us/previous-versions/technet-magazine/cc462815(v=msdn.10))
But, Could we view the icon of 16bit executive again?

**Comment by cracyc** [comment]
https://github.com/otya128/Icon16bitFix
